doc/devguide/codebase/code-style.rst

```diff
@@ -6,12 +6,165 @@ Suricata uses a fairly strict coding style. This document describes it.
 Formatting
 ~~~~~~~~~~
 
+clang-format
```

Comment:
If the diff sucks, see https://github.com/roligugus/suricata/blob/clang-format-proposal-v4/doc/devguide/codebase/code-style.rst for how it's displayed